x-pack/plugins/observability_solution/profiling/public/components/topn_functions/index.tsx

```diff
@@ -50,6 +51,7 @@ export const TopNFunctionsGrid = ({
   topNFunctions,
   comparisonTopNFunctions,
   totalSeconds,
+  showFullScreenSelector = false,
```

Comment:
Perhaps it's better to define the default `true` to to affect existing consumers? Unless this is only being used in a single place.

Reply:
Done ✅